packages/main/test/specs/MultiComboBox.spec.js

```diff
@@ -812,6 +812,96 @@ describe("MultiComboBox general interaction", () => {
 
 			assert.strictEqual(await mcb.getProperty("value"), "", "Input field is empty");
 		});
+
+		it ("F4 should focus the selected item or the first one if there is no selected", async () => {
```

Comment:
Maybe, I have a problem not the change itself, but couple tests are failing locally on my machine. May I ask you to run them once again in order to compare the results as I noticed that the build was successful